src/components/dialog-test.js

```diff
@@ -99,6 +100,19 @@ describes.realWin('Dialog', {}, env => {
       expect(graypaneStubs.show).to.not.be.called;
     });
 
+    it('should open dialog as hidden', function* () {
+      immediate();
+      dialog.open(ANIMATE, HIDDEN);
+      yield dialog.animating_;
```

Comment:
Shouldn't we ensure/test to confirm that it definitely is not animating when hidden?

Reply:
need to address.